CP1404_practicals/prac_05/emails.py

```diff
@@ -28,4 +25,11 @@ def get_name_from_email(user_email):
     return name
 
 
+# align the emails so they are in one nice column
```

Comment:
Since this comment explains what a function does, it could instead be written as the function docstring on line 30.